.github/workflows/test_linux_packages.yml

```
@@ -0,0 +1,72 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test Linux Packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - ADHOCBUILD
+  workflow_call:
+
+jobs:
+  test_rocm_info:
+    name: "Sanity ROCM Test"
+    runs-on: linux-mi300-1gpu-ossci
+    strategy:
+      fail-fast: true
+    defaults:
+      run:
+        shell: bash
+    env:
+      VENV_DIR: ${{ github.workspace }}/.venv
+
+    # MOSTLY BOILER PLATE ABOVE.
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
```

```suggestion
        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```
I recently pinned all jobs to a specific version which then get updated with Dependabot.